test/model-property.js

```diff
@@ -9,10 +9,12 @@ var ModelProperty = app.models.ModelProperty;
 var ModelDefinition = app.models.ModelDefinition;
 var ConfigFile = app.models.ConfigFile;
 var TestDataBuilder = require('./helpers/test-data-builder');
+var request = require('supertest');
 
 describe('ModelProperty', function() {
   beforeEach(givenBasicWorkspace);
   beforeEach(function(done) {
+    this.modelId = 'server.user';
```

Comment:
Good point. `this.modelId` will store the data on a "context" object passed as `this` to all Mocha test/hook callbacks. We could use function scopes and `var modelId` instead, that would work equally well here.

I used `this.modelId` for consistency with the existing code around.